lightwood/helpers/general.py

```diff
@@ -146,11 +146,6 @@ def evaluate_array_accuracy(
             if ts_analysis['tss'].nr_predictions == 1:
                 preds = np.expand_dims(preds, axis=1)
 
-            # only evaluate accuracy for rows with complete historical context
```

Comment:
how come this is omitted now?

Reply:
I thought this hadn't made the push, my bad. Regardless, I'm in the process of finishing a change so that ignoring these rows is now a burden on 1) the mixers and 2) the accuracy functions. Why? Because otherwise, we lose valuable data in the transform process that this mixer (and potentially others in the future) will absolutely want to use.

I promise the change isn't very big :D keep you posted